connect/connectNotificationServer.js

```diff
@@ -307,6 +335,7 @@ const handler = (topic, message, logger, callback) => {
       //       - check that event has everything required or throw error
       getNotificationsForTopicStarter(eventConfig, message.topicId),
```

Comment:
@gondzo we might reuse the logic of `originator`, which I mentioned in other comment, here to be generic enough to send notifications to the originator.